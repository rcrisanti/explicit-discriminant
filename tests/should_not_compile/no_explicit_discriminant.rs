use explicit_discriminant::ExplicitDiscriminant;

#[derive(ExplicitDiscriminant)]
enum Test {
    One = 1,
    Two = 2,
    Three,
    Four = 4,
}

fn main() {
    let t = Test::One;
}
