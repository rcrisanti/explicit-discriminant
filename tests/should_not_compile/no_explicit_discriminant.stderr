error: no explicit discriminant
 --> tests/should_not_compile/no_explicit_discriminant.rs:7:5
  |
7 |     Three,
  |     ^^^^^
